FirebaseRemoteConfig/Sources/RCNConfigFetch.m

```diff
@@ -577,4 +584,15 @@ - (NSURLSessionDataTask *)URLSessionDataTaskWithContent:(NSData *)content
   return [_fetchSession dataTaskWithRequest:URLRequest completionHandler:fetcherCompletion];
 }
 
+- (NSString *)getTemplateVersionNumber {
+  if (_content.fetchedConfig != nil &&
+      [_content.fetchedConfig objectForKey:templateVersionNumberKey] &&
+      ![[_content.fetchedConfig objectForKey:templateVersionNumberKey]
+          isKindOfClass:[NSNull class]]) {
```

Comment:
I will like [_content.fetchedConfig objectForKey:templateVersionNumberKey] will be able to detect if it's null so 
![[_content.fetchedConfig objectForKey:templateVersionNumberKey]
          isKindOfClass:[NSNull class]]
is not needed.

you might want to check sth like 

[[_content.fetchedConfig objectForKey:templateVersionNumberKey]
          isKindOfClass:[NSString class]] or NSNumber depends on how you store it.

Reply:
Updated